### ISSUE TITLE
amplify init returns could not initialize error

### ISSUE THREAD

**Issue by pstaylor-patrick** [aws-amplify/amplify-cli#10103]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### How did you install the Amplify CLI?

npm i -g @aws-amplify/cli

### If applicable, what version of Node.js are you using?

nove v16.13.1

### Amplify CLI Version

amplify v7.6.26

### What operating system are you using?

macOS Monterey v12.3 on a 2020 MacBook Pro (Apple M1)

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

no

### Amplify Categories

Not applicable

### Amplify Commands

init, pull, push

### Describe the bug

I'm experiencing an AWS Amplify outage in the aws cli, the amplify cli, and the aws console. I started to see this behavior yesterday afternoon (Wed Mar 30, 2022 around 2 pm Central).

As far as the amplify cli is concerned, this is what I see when running `amplify init`:

> Could not initialize 'kngdmprod': The difference between the request time and the current time is too large.

And this is what I see when running `aws amplify list-apps`:

> An error occurred (InvalidSignatureException) when calling the ListApps operation: Signature expired: 20220331T093848Z is now earlier than 20220331T094912Z (20220331T095412Z - 5 min.)

And this is what I see in the AWS Amplify Console:

> List apps call failed
> Network Failure

<img width="1233" alt="Screen Shot 2022-03-31 at 04 43 45" src="https://user-images.githubusercontent.com/1963845/161031698-20bd739f-27ff-4ea7-b117-388ed7963bda.png">

### Expected behavior

I expect to see and have access to my amplify apps.

### Reproduction steps

You can either:
1. `amplify init`
2. `aws amplify list-apps`
3. <https://us-east-1.console.aws.amazon.com/amplify/home?region=us-east-1#/>

### Log output

<details>

```
# Put your logs below this line

2022-03-30T15:21:48.209Z|info : amplify status core  {"yes":false}
2022-03-30T15:21:48.243Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:23:56.548Z|info : amplify remove storage  
2022-03-30T19:23:56.586Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:24:04.114Z|info : amplify update function  
2022-03-30T19:24:04.150Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:24:18.203Z|info : amplify remove storage  
2022-03-30T19:24:18.239Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:24:28.438Z|info : amplify add storage  
2022-03-30T19:24:28.474Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:24:55.909Z|info : amplify add custom  
2022-03-30T19:24:55.946Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:25:10.066Z|info : amplify add function  
2022-03-30T19:25:10.101Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:26:24.618Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:26:24.620Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:26:24.620Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:26:24.624Z|info : amplify-provider-awscloudformation.aws-lambda.Lambda.listLayerVersions([{"LayerName":"[***]ossdk-[***]prod","MaxItems":20}])
2022-03-30T19:26:41.448Z|info : amplify add function  
2022-03-30T19:26:41.484Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:27:25.163Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:27:25.164Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:27:25.164Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:27:25.167Z|info : amplify-provider-awscloudformation.aws-lambda.Lambda.listLayerVersions([{"LayerName":"[***]ossdk-[***]prod","MaxItems":20}])
2022-03-30T19:27:29.495Z|info : amplify status core  {"yes":false}
2022-03-30T19:27:29.532Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:27:40.067Z|info : amplify add storage  
2022-03-30T19:27:40.103Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:28:14.121Z|info : amplify add custom  
2022-03-30T19:28:14.158Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:29:42.494Z|info : amplify status core  {"yes":false}
2022-03-30T19:29:42.530Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:30:11.751Z|info : amplify update api  
2022-03-30T19:30:11.788Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:30:33.118Z|info : amplify update api  
2022-03-30T19:30:33.185Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:30:40.192Z|info : amplify update api  
2022-03-30T19:30:40.229Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:30:48.843Z|info : amplify update api  
2022-03-30T19:30:48.880Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:30:57.286Z|info : amplify remove custom  
2022-03-30T19:30:57.321Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:31:04.629Z|info : amplify remove storage  
2022-03-30T19:31:04.665Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:31:10.885Z|info : amplify status core  {"yes":false}
2022-03-30T19:31:10.921Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:31:26.185Z|info : amplify add custom  
2022-03-30T19:31:26.220Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:31:47.720Z|info : amplify add function  
2022-03-30T19:31:47.755Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:32:23.223Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:32:23.224Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:32:23.224Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:32:23.227Z|info : amplify-provider-awscloudformation.aws-lambda.Lambda.listLayerVersions([{"LayerName":"[***]ossdk-[***]prod","MaxItems":20}])
2022-03-30T19:32:26.140Z|info : amplify add function  
2022-03-30T19:32:26.175Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:34:00.980Z|info : amplify push core  {"yes":true}
2022-03-30T19:34:01.018Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:34:01.837Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:34:01.838Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:34:01.838Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:34:01.843Z|info : amplify-provider-awscloudformation.aws-s3.deleteDirectory.s3.getAllObjectVersions([{"BucketName":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
2022-03-30T19:34:01.843Z|info : amplify-provider-awscloudformation.aws-s3.getAllObjectKey.s3.listObjectVersions([{"Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment","Prefix":"hooks/"}])
2022-03-30T19:36:03.081Z|info : amplify push core  {"yes":true}
2022-03-30T19:36:03.118Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:36:04.447Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:36:04.448Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:36:04.448Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:36:04.452Z|info : amplify-provider-awscloudformation.aws-s3.deleteDirectory.s3.getAllObjectVersions([{"BucketName":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
2022-03-30T19:36:04.453Z|info : amplify-provider-awscloudformation.aws-s3.getAllObjectKey.s3.listObjectVersions([{"Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment","Prefix":"hooks/"}])
2022-03-30T19:37:10.538Z|info : amplify push core  {"yes":true}
2022-03-30T19:37:10.574Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:37:11.389Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:37:11.389Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:37:11.389Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:37:11.393Z|info : amplify-provider-awscloudformation.aws-s3.deleteDirectory.s3.getAllObjectVersions([{"BucketName":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
2022-03-30T19:37:11.393Z|info : amplify-provider-awscloudformation.aws-s3.getAllObjectKey.s3.listObjectVersions([{"Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment","Prefix":"hooks/"}])
2022-03-30T19:39:12.571Z|info : amplify status core  {"yes":false}
2022-03-30T19:39:12.609Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:39:40.420Z|info : amplify push core  {"yes":true}
2022-03-30T19:39:40.457Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:39:42.203Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:39:42.203Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:39:42.203Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:39:42.209Z|info : amplify-provider-awscloudformation.aws-s3.deleteDirectory.s3.getAllObjectVersions([{"BucketName":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
2022-03-30T19:39:42.209Z|info : amplify-provider-awscloudformation.aws-s3.getAllObjectKey.s3.listObjectVersions([{"Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment","Prefix":"hooks/"}])
2022-03-30T19:39:50.968Z|info : amplify status core  {"yes":false}
2022-03-30T19:39:51.007Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:40:03.765Z|info : amplify init core  
2022-03-30T19:40:03.802Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:40:09.219Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:40:09.220Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:40:09.220Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:40:09.228Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:40:09.228Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:40:09.228Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:40:09.232Z|info : amplify-provider-awscloudformation.zip-util.downloadZip.s3.getFile([{"Key":"[***]rent-[***]ud-[***]d.zip"},null])
2022-03-30T19:40:09.233Z|info : amplify-provider-awscloudformation.aws-s3.s3.getFile([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
2022-03-30T19:40:10.709Z|error : amplify-provider-awscloudformation.aws-s3.s3.getFile([{"Key":"[***]rent-[***]ud-[***]d.zip","Bucket":"[***]ify-[***]ogos-[***]prod-[***]152-[***]ment"}])
RequestTimeTooSkewed: The difference between the request time and the current time is too large.
2022-03-30T19:40:19.609Z|info : amplify pull core  
2022-03-30T19:40:19.643Z|info : amplify-cli-core.banner-message/index.ts.fetch banner messages from https://aws-amplify.github.io/amplify-cli/banner-message.json({}
2022-03-30T19:40:23.162Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileConfig(["kngdm-amplify"])
2022-03-30T19:40:23.164Z|info : amplify-provider-awscloudformation.system-config-manager.getProfiledAwsConfig.profileConfig([{"region":"us-east-1","output":"json"}])
2022-03-30T19:40:23.164Z|info : amplify-provider-awscloudformation.system-config-manager.getProfileCredentials(["kngdm-amplify"])
2022-03-30T19:40:23.173Z|info : amplify-provider-awscloudformation.amplify-service-permission-check.checkAmplifyServiceIAMPermission.amplifyClient.listApps([])
2022-03-30T19:40:26.898Z|error : amplify-provider-awscloudformation.amplify-service-permission-check.checkAmplifyServiceIAMPermission.amplifyClient.listApps([])
InvalidSignatureException: Signature expired: 20220330T194025Z is now earlier than 20220330T195049Z (20220330T195549Z - 5 min.)
```

</details>

### Additional information

related to https://github.com/aws/aws-cli/issues/6833

**Comment by pstaylor-patrick** [comment]
✅ I just resolved this issue on my own.

⏱ I discovered that my laptop time was off by more than 10 minutes.

👀 I realized this after comparing to the time on my MacBook with the times on my iPhone, wristwatch, and desk clock.

🤷‍♂️ I don't know how the laptop time originally drifted from reality, but all of the related AWS issues were immediately resolved after I reset my time clock.

🔧 I just went into the macOS System Preferences app and toggled the automatic time detection off and back on again to reset the time.

<img width="780" alt="Screen Shot 2022-03-31 at 09 53 16" src="https://user-images.githubusercontent.com/1963845/161085296-de6bf59f-d326-4bea-b676-26a94310dc52.png">
